CMakeLists.txt

```diff
@@ -692,7 +692,7 @@ if(BUILD_TESTS)
       NAME code_update_test
       PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/code_update.py
       CONSENSUS cft
-      ADDITIONAL_ARGS --oe-binary ${OE_BINDIR} --raft-election-timeout 20000
+      ADDITIONAL_ARGS --oe-binary ${OE_BINDIR} --raft-election-timeout 4000
```

Comment:
This seemed very high. Let's see if this still works and if not, let's find out why not.

Reply:
This is deliberately high because we've historically suffered from unwanted elections caused by very slow quote validation. With recent DCAP cache changes, it's been less of a problem.